lib/kind/__lib__/strict.rb

```diff
@@ -53,6 +53,8 @@ def in!(list, value)
     end
 
     def assert_hash!(hash, options)
+      raise ArgumentError, "hash cannot be empty" if hash.empty?
```

Comment:
@haroldofurtado  related to https://github.com/serradura/kind/pull/60#discussion_r659851107